ppd/ppd.h

```diff
@@ -880,13 +880,33 @@ extern cf_cmyk_t	*ppdCMYKLoad(ppd_file_t *ppd,
 
 // **** New in libppd 2.0.0: Function to check the correctness
 //      of PPD files, originally cupstestppd ****
-extern cups_array_t	*ppdTest(int ignore_pc_filenames, int ignore_filters,
-                  int ignore_profiles, int ignore_none, int ignore_all, char *rootdir,
-                  int warn_none, int warn_constraints, int warn_defaults,
-                  int warn_duplex, int warn_filters, int warn_profiles, int warn_sizes,
-                  int warn_translations, int warn_all, int help, int verbose, int relaxed,
-                  int q_with_v, int v_with_q, int root_present, int files,
-                  cups_array_t *file_array, cups_array_t *stdin_array);
+
+typedef struct ignore_parameters_s
```

Comment:
Please use real bit fields here, not structs. bitfields are integer variables where each bit represents one boolean state and the bits are accessed via bit-wise AND and OR operations. Use the bitfields which the `ppdTest()` function is suing internally also as parameter for it and let `main()` of `testppdfile.c` do the `|=` to add the bits according to the user-supplied command line arguments.